src/commands/contract/compile.ts

```diff
@@ -16,6 +16,12 @@ export class CompileContract extends SwankyCommand<typeof CompileContract> {
       description:
         "A production contract should always be build in `release` mode for building optimized wasm",
     }),
+    verifiable: Flags.boolean({
+      default: false,
+      char: "v",
+      description:
+        "A production contract should be build in `verifiable` mode to deploy on a public network",
```

Comment:
Maybe we can extend the description by mentioning: 'Ensure Docker Engine is up and running.'